setupMYM.m

```diff
@@ -0,0 +1,67 @@
+function setupMYM(version, force)
```

Comment:
```suggestion
function mymVersion = setupMYM(version, force)
```
Would be nice if calls to this returned the installed version. Could also make sense to apply to `setupDJ`.

Reply:
can see returning version is useful, renaming function I'm less clear on since this doesn't simply return the version but also may modify the system

Reply:
No need to rename the function but the way in Matlab that you return arguments is by defining a variable to output result to. Then you simply set that within your method. Such as `mymVersion`.

Reply:
ah, misread this. what should we return? string? 3-array of versions?

Reply:
Hmm don't remember our convention at the moment but I'd likely just stick to how version is returned in the `dj.version` function.

Reply:
so, version support is partially implemented - some issues w/r/t proposed scheme ('how dj.version works') below -
current approach applies the input argument (string) to the corresponding file, and if mymSetup succeeds, returns this value back to the user, and replaces the INVOKED variable with INSTALLED_VERSION to cache this for subsequent non-force calls. Paused here as anything further needs wider discussions / more intrusive changes.

so - to the issues on 'how dj version works':

1) This function has to support new abstraction of 'master' (if I install 'master', should I get 'master' back or '2.3.4' ? if 2.3.4 where is master:2.3.4 reference mapping stored for confirmation/tests/maintenance, etc). For now, returning as specified in input argument if install succeeded (see point 2). setupDJ doesn't suffer from this problem because it doesn't need to manage other files / take any version arguments / etc

2) mym('version') and dj.version() both return 'struct with fields', which isn't how we are expecting the input argument, which makes version checking of input arg vs result tricky even without the master:version mapping problem in point 1

3) mym/mymSetup doesn't directly return anything, so the only way to get the 'real' mym version is mym('version') after calling mymSetup(), not sure if there could be issues here w/r/t some hidden mym internal state / install sequence problems.

also related to the unit test: unit test is clearing various directories but calling back into setupMYM() without force, which means unit test environment is unreliable. not sure how best to handle this.

Reply:
Thanks for the update. I will provide feedback on these points in my review where they associate within the code.